app/plugins/prometheus.md

```diff
@@ -0,0 +1,183 @@
+---
+id: page-plugin
+title: Plugins - Prometheus
+header_title: Prometheus
+header_icon: https://konghq.com/wp-content/uploads/2018/06/prometheus.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Usage
+    items:
+      - label: Log Format
+description: |
+    Expose metrics related to Kong and the upstream services in [Prometheus](https://prometheus.io/docs/introduction/overview/) Exposition format which can be scrapped by a Prometheus Server.
+
+params:
+  name: prometheus
+  api_id: true
+  service_id: true
+  route_id: true
+
+---
+
+### Reading metrics into Prometheus
+
+Metrics are availble on the Admin API at `http://localhost:8001/metrics`
+endpoint. Note the URL to the Admin API will be specific to your installation.
+
+
+#### Available metrics
+
+- *Status codes*: HTTP status codes returned by upstream services. 
+  These are available per service and across all services.
+- *Latencies Histograms*: Latency as measured at Kong:
+   - *Request*: Total request latency 
+   - *Kong*: Time taken for Kong to route, authenticate and run all plugins for a request
+   - *Upstream*: Time taken by the upstream to respond to the request.
+- *Bandwidth*: Total Bandwidth (egress/ingress) flowing through Kong.
+  This metrics is availble per service and across all services.
+- *DB reachability*: Can the Kong node reach it's Database or not (Guage 0/1).
+- *Connections*: Various NGINX connection metrics like active, reading, writing,
+  accepted connections.
+
+Here is an example of output you could expect from the `/metrics` endpoint:
+
+```bash
+$ curl http://localhost:8001/metrics
+root@vagrant-ubuntu-trusty-64:~# curl -D - http://localhost:8001/metrics
```

Comment:
It seems like this line can go away since the line above this has the prompt and command.

Reply:
Done.